SaraAlert/SaraAlert
# SARAALERT-1694: Upload User Documentation to Github

# Description
Jira Ticket: [SARAALERT-1694](https://tracker.codev.mitre.org/browse/SARAALERT-1694)

Adds a user-materials directory and includes all of the documentation linked in [SARAALERT-1694](https://tracker.codev.mitre.org/browse/SARAALERT-1694) to user-materials. Track resources in `git lfs`.

## (Feature) Demo/Screenshots
N/A

## Important Changes
`user-materials/`
- All 19 documentation files added to `user-materials/`
- User guides added to `user-guides` subfolder
- Quick start guides added to `quick-start-guides` subfolder

## Testing Recommendations
- Check `.gitattributes` and ensure that folder is tracked correctly. 
- Check that all linked documentation is in the `user-materials/` folder. 

# Checklists

**Submitter:**


@tstrass  :


@timwongj  :


@hackrm  :
